<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Spravka_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107,92,209,220); color: White; border-radius: 20px; </string>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Menu_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107,92,209,200); color: White; border-radius: 20px; </string>
   </property>
   <property name="text">
    <string>Меню</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Profile_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107,92,209,150); color: White; border-radius: 20px; </string>
   </property>
   <property name="text">
    <string>Профиль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_btn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107,92,209,150); color: White; border-radius: 20px; </string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
